md.4

```diff
@@ -940,8 +940,7 @@ After crashes, if one of the not modified data disks of
 the stripe is missing, this updated parity can be used to recover its
 data.
 
-This mechanism is documented more fully in the file
-Documentation/md/raid5-ppl.rst
+This mechanism is documented more fully in the file Documentation/driver-api/md/raid5-ppl.rst
```

Comment:
I'd preffer "in detail" rather than "more fully" :)

Reply:
Ok, you forced me to look closer into it.